src/test/java/rest/gorestcoIn/APIRest.java

```diff
@@ -0,0 +1,63 @@
+package rest.gorestcoIn;
+
+import io.restassured.RestAssured;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import rest.gorestcoIn.utils.User;
+import rest.gorestcoIn.utils.helpermethods.APICal;
+
+public class APIRest{
+
+    @BeforeClass
+    public void setup() {
+        RestAssured.baseURI = "https://gorest.co.in/";
+        RestAssured.basePath = "public-api/";
+
+    }
+
+    @Test
+    public void postUser() {
+
+        Header header = new Header("Authorization", "Bearer 73e824b7f681b6e4d47c97ed07c2e26f6f91effd9a3496aad4f4960798709544");
```

Comment:
What is the aim to create header in test layer, move it at least to class/layer responsible to do API calls